lib/generators/cache/database/templates/add_cache_compression_column.rb

```diff
@@ -0,0 +1,5 @@
+class AddCacheCompressionColumn < ActiveRecord::Migration[5.2]
+  def change
+    add_column :activesupport_cache_entries, :compression, :string, null: false, default: 'plain'
```

Comment:
this should be nullable, no need to store `'plain'` for every row, it's a waste of table space